packages/jest-runtime/src/__tests__/transform-test.js

```diff
@@ -157,41 +158,39 @@ describe('transform', () => {
     expect(fs.readFileSync).toBeCalledWith('/fruits/banana.js', 'utf8');
 
     // in-memory cache
-    const response2 = transform('/fruits/banana.js', config).script;
+    const response2 = scriptTransformer.transform('/fruits/banana.js').script;
     expect(response2).toBe(response);
 
```

Comment:
We can assume if we have referential equality, that we didn't go to the filesystem and create another vm.Script instance.